### ISSUE TITLE
Zone type Analysis: Run statistical tests to verify the density grouping results

### ISSUE THREAD

**Issue by manaswis** [ProjectSidewalk/sidewalk-data-analysis#31]
For section 4.5.2, we need to test the statistical significance of the results showing that accuracies across the density based groups are similar.

**Comment by misaugstad** [comment]
I don't think that we need a statistical test for this. A statistical test to say that the means are truly "similar" is actually really uncommon and not simple. Just testing to see if there is a difference between the means and failing to reject the null (the null being that the means are equal) is not sufficient to say that the means _are_ similar. This really feels like one of those places where we can just let the data speak for themselves and we don't need a statistical test; it opens up this whole other can of statistical worms that I don't think is worth opening (the can contains things like [Bland-Altman plots](https://en.wikipedia.org/wiki/Bland%E2%80%93Altman_plot) and [estimation statistics](https://en.wikipedia.org/wiki/Estimation_statistics))

**Comment by misaugstad** [comment]
@manaswis thoughts?

**Comment by manaswis** [comment]
I will wait until I can confirm with @jonfroehlich about this. Obviously, this would require reading the sections. Hence, I would hold off on it until then.

**Comment by manaswis** [comment]
I have another idea for zone type analysis, which might show differences in accuracies. 

I went through the definitions of the zones again. Based on the definitions of the given zone types, instead of two groups of densities, we should have low, moderate, medium and high density groups. We might get better results. I am sure the built infrastructure is different in zones with different densities. I think we just have to figure out the correct sub-groups for the zones, and I think a finer grouping based on density should work. 

@misaugstad: Do you think we can divide routes based on these finer groups? Is it possible?

**Comment by misaugstad** [comment]
>Do you think we can divide routes based on these finer groups? Is it possible?

I don't think so. When I tried to do the density analysis originally, that is what I wanted to do, but there is just not enough data in the high density range. I don't remember exactly how much there was, but I did try it before and saw that there wasn't enough.

**Comment by manaswis** [comment]
If there are significant sample sizes in other groups, I think it should be fine if there isn't much in one group. We won't analyse it. At least we would have three distinct groups. I am suggesting this because I was reading about sidewalk planning and density measures that they have for land use planning, the infrastructure is built differently based on the density of the land use type.

**Comment by misaugstad** [comment]
Yeah, unknown and honestly it is more likely to be a very low ROI, since the interesting zones are still likely to be the ones with not enough data :)

**Comment by jonfroehlich** [comment]
Is there another data source with density information for DC

**Comment by misaugstad** [comment]
>Is there another data source with density information for DC

I don't think there is another source that really gets us what we are looking for... We could get population density by census tract, but I don't think that's precisely what we want. For example, commercial areas may have low population density, and I think we want those labeled as high density.

**Comment by misaugstad** [comment]
>So it is a form of population density

I still don't think I entirely agree with that. Again, commercial areas may have a high floor area ratio, but relatively few people who actually live there. So I don't think it is really a form of population density.

**Comment by manaswis** [comment]
Which is why I said it is a form of population density. By definition, it is the number of people per square mile of land area. In a broader sense, it looks at the number of people living in a square mile.

The density measures used for zoning in relation to sidewalks relates to the incidence of people and automobiles in the land area.

**Comment by misaugstad** [comment]
@manaswis do you think the section works without having some test of similarity?

**Comment by manaswis** [comment]
Hold off on this now. Will depend on what we end up writing.
